src/DataFactory/DataFactoryV2.Test/ScenarioTests/IntegrationRuntimeTests.ps1

```diff
@@ -64,6 +64,12 @@ function Test-SelfHosted-IntegrationRuntime
             -Force
         Assert-AreEqual $result.Description $description
 
+        $status = Get-AzDataFactoryV2IntegrationRuntime -ResourceGroupName $rgname `
+            -DataFactoryName $dfname `
+            -Name $irname `
+            -Status
+        Assert-NotNull $status.LatestVersion
+
```

Comment:
If you change the scenario tests, you will need to re-record the flow, following [recordingrunning-tests](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/azure-powershell-developer-guide.md#recordingrunning-tests)

Reply:
Updated the codes.